### DISCUSSION TITLE
WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED! after upgrading to 15.3.5

### DISCUSSION THREAD

**Discussion by lfb-sanger** [gravitational/teleport#41691] [Q&A] [answered]
Hi,

We have recently upgrade to 15.3.5 after seeing the Teleport Security Release - May 2024 email and we are now seeing the message below each time we login using SSH. When we see the message we clear the host from our know_host file and we can login once, but when we try again we get the same message. It seems to be affecting everyone who is using SSH. Using tsh ssh is working. We have used tsh config as the base for our .ssh/config file.

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
Someone could be eavesdropping on you right now (man-in-the-middle attack)!
It is also possible that a host key has just been changed.
The fingerprint for the RSA key sent by the remote host is
\<fingerprint\>
Please contact your system administrator.
Add correct host key in /Users/\<user\>/.ssh/teleport/known_hosts to get rid of this message.
Offending RSA key in /Users/\<user\>/.ssh/teleport/known_hosts:1
Host key for [\<hostname\>]:3022 has changed and you have requested strict checking.
Host key verification failed.

Is there anything we would need to change in our .ssh/config file to resolve this?

Any help appreciated.

Thank you,
Luke

**Comment by lfb-sanger** [comment, marked as answer]
We have found the issue and are publishing a change to the .ssh config file for our users.

**Reply by zmb3** [reply]
Glad to hear it.

For future reference, Teleport Enterprise customers with questions about private security releases should submit a support ticket or reply directly to the security notification rather than asking in public forums like this one.

**Reply by lfb-sanger** [reply]
@zmb3 sorry about that I will in future. Thank you.
